crc32.c

```
@@ -10,35 +10,61 @@
 #include "functable.h"
 #include "crc32_braid_tbl.h"
+#include "crc32_p.h"
 
 /* ========================================================================= */
 
-const uint32_t * Z_EXPORT PREFIX(get_crc_table)(void) {
+Z_EXPORT const uint32_t * PREFIX(get_crc_table)(void) {
     return (const uint32_t *)crc_table;
 }
 
+Z_INTERNAL uint32_t crc32_small(uint32_t crc, const unsigned char *buf, size_t len) {
```

Cppcheck (2.20.0)

[style] The function 'crc32_small' is never used.

(unusedFunction)